### PR TITLE
A+C: update email Lyron Winderbaum

### PR DESCRIPTION
**Pull request by Armadilloa16**
Update email in A+C.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -63,7 +63,7 @@ Kevin C. Zimmerman <kevinczimmerman@gmail.com>
 Kirill Motkov <motkov.kirill@gmail.com>
 Konstantin Shaposhnikov <k.shaposhnikov@gmail.com>
 Leonid Kneller <recondite.matter@gmail.com>
-Lyron Winderbaum <lyron.winderbaum@student.adelaide.edu.au>
+Lyron Winderbaum <armadilloa16@gmail.com> <lyron.winderbaum@uwa.edu.au>
```
Comment (kortschak) [inline AUTHORS:66]: Leave the Adelaide address here. It marks commits made by that user. Same in CONTRIBUTORS. If your commits are associated with the gmail address, you don't need to add the uwa address.

Comment (Armadilloa16) [reply AUTHORS:66]: Oooh ok, there isn't a way to say "don't try emailing me here"?

Comment (kortschak) [reply AUTHORS:66]: Yes, they can email you there and get a bounce. This is really a contact list, it's a look-up table of authorships/contributors.

Comment (Armadilloa16) [reply AUTHORS]: Ok, makes sense, I'll just include all emails then.

File: CONTRIBUTORS [hunk 2]
```diff
@@ -71,7 +71,7 @@ Kevin C. Zimmerman <kevinczimmerman@gmail.com>
 Kirill Motkov <motkov.kirill@gmail.com>
 Konstantin Shaposhnikov <k.shaposhnikov@gmail.com>
 Leonid Kneller <recondite.matter@gmail.com>
-Lyron Winderbaum <lyron.winderbaum@student.adelaide.edu.au>
+Lyron Winderbaum <lyron.winderbaum@student.adelaide.edu.au> <armadilloa16@gmail.com>  <lyron.winderbaum@uwa.edu.au>
```
Comment (kortschak) [inline CONTRIBUTORS:74]: Double space between gmail and uwa address.

Comment (Armadilloa16) [reply CONTRIBUTORS]: ... this is why I love go fmt

**Review by kortschak — APPROVED** [review]

File: CONTRIBUTORS (hunk 2 above)
Comment (kortschak) [reply CONTRIBUTORS]: Indeed

**Pull request merged**